docs/HTTP_API.md

```diff
@@ -1705,6 +1791,91 @@ The handler-specific descriptions below describe how each handler populates the
     {"meta":{"status":"OK","type":"application/json"},"data":{result:[{"name":"Name","description":"Recording name","defaultValue":"Recording"},{"name":"Duration","description":"Duration of recording","defaultValue":"30s[s]"},{"name":"Max Size","description":"Maximum size of recording","defaultValue":"0B[B]"},{"name":"Max Age","description":"Maximum age of the events in the recording","defaultValue":"0s[s]"},{"name":"To disk","description":"Record to disk","defaultValue":"false"},{"name":"Dump on Exit","description":"Dump recording data to disk on JVM exit","defaultValue":"false"}]}}
     ```
 
+* #### `TargetRecordingGetHandler`
+
+    ###### synopsis
+    Returns a recording of a target JVM as an octet stream, using a JWT for
+    authorization. See `AuthTokenPostHandler`.
```

Comment:
I guess we could add a quick link back to `AuthTokenPostHandler` section? Same for others that have `See [handler]` clause?